.gitignore

```diff
@@ -2,5 +2,5 @@
 .Rhistory
 .RData
 .Ruserdata
-
 _targets/*
+*/out/*
```

Comment:
Minor comment - its best practice to have a blank line at the end of the .gitignore file